app/params/proto.go

```diff
@@ -1,3 +1,4 @@
+//go:build !test_amino
 // +build !test_amino
```

Comment:
cref https://pkg.go.dev/cmd/go#hdr-Build_constraints

I didn't know about this changing in 1.17 until now